arex-instrumentation/servlet/arex-httpservlet/src/main/java/io/arex/inst/httpservlet/convert/BodyConverters.java

```diff
@@ -0,0 +1,10 @@
+package io.arex.inst.httpservlet.convert;
+
+import io.arex.inst.httpservlet.adapter.ServletAdapter;
+
+public interface BodyConverters<HttpServletRequest, HttpServletResponse> {
+
+    boolean match(HttpServletRequest request, ServletAdapter adapter);
+
+    String getBody(HttpServletRequest request, ServletAdapter adapter);
+}
```

Comment:
- Because the current parsing is processed through content type, in order to prevent other parameter judgments (such as httpmethod) in the later stage, I passed adapter and request in the match.
- Can support modifying return values to byte arrays
- Can support increasing response of return values


So my idea is whether to keep the match as it is and whether to consider adding responses (such as file downloads)

Reply:
> * Because the current parsing is processed through content type, in order to prevent other parameter judgments (such as httpmethod) in the later stage, I passed adapter and request in the match.

Agree with you.

> * Can support modifying return values to byte arrays
> * Can support increasing response of return values
> 
> So my idea is whether to keep the match as it is and whether to consider adding responses (such as file downloads)

`file downloads` No support

Reply:
I have adjusted the match method to include the logic of the response, so I hope it can also cover the logic judgment of the response currently